Source/SwiftLintBuiltInRules/Rules/Lint/UnusedImportRuleExamples.swift

```diff
@@ -122,7 +125,7 @@ struct UnusedImportRuleExamples {
             dispatchMain()
             """),
         Example("""
-        ↓@_exported import Foundation
+        ↓@_implementationOnly import Foundation
```

Comment:
The original intent in https://github.com/realm/SwiftLint/pull/2993 seemed to make sure _any_ attribute was correctable, so I arbitrarily chose `@_implementationOnly` as a replacement.